test/index.js

```diff
@@ -18,6 +18,26 @@ suite(`${pkg.name} unit`, () => {
 
     assert.deepStrictEqual(next.toISO(), DateTime.fromISO('2020-04-16T13:00:00.0Z').toISO())
   })
+
+  test('process schedule with zone', () => {
+    const next = meetings.getNextScheduledMeeting(
+      ['2020-04-02T17:00:00/P7D'],
+      DateTime.fromISO('2021-01-01T00:00:00.0Z'),
+      'America/Los_Angeles'
+    )
+
+    assert.deepStrictEqual(next.toISO(), DateTime.fromISO('2020-12-31T16:00:00.000-08:00').toISO())
```

Comment:
Actually, the goal here should be to keep it as `17:00`, this PR needs work 😓

Reply:
Fixed, the issue was `P7D` was being converted due to `Interval` into something with less specificity (e.g. `milliseconds`) and that did not work when iterating forward through expectations here. Splitting manually fixed this.